.gitignore

```diff
@@ -1 +1 @@
-node_modules/
+node_modules
```

Comment:
@geraldhumphries sorry for being picky but can you please remove all the changes to the `.gitignore`. After this I'll be happy to merge it in. Thanks for your work!